iis-wf/iis-wf-referenceextraction/src/main/resources/eu/dnetlib/iis/wf/referenceextraction/patent/main/oozie_app/workflow.xml

```diff
@@ -124,6 +219,111 @@
                 </property>
             </configuration>
         </sub-workflow>
+        <ok to="metadata_retriever_input_transformer"/>
+        <error to="fail"/>
+    </action>
+
+    <action name="metadata_retriever_input_transformer">
+        <spark xmlns="uri:oozie:spark-action:0.2">
+            <prepare>
+                <delete path="${workingDir}/metadata_retriever_input"/>
+            </prepare>
```

Comment:
You are right, those `<prepare>` entries are redundant with the spark code. I have blindly followed the other wf definitions where we are cleaning the output directories in both spark code and wf definition.

Since this is the problem in many other workflow definitions I have created a ticket to get rid of such redundancy: #1102.